lib/general.js

```diff
@@ -0,0 +1,47 @@
+import { ok, nok } from './utils';
+import { exec } from 'teen_process';
+import { DoctorCheck } from './doctor';
+import NodeDetector from './node-detector';
+
+
+let checks = [];
+
+// Node Binary
+class NodeBinaryCheck extends DoctorCheck {
```

Comment:
Yes. Previously if you only ran Android check it wouldn't get run.

But yes, it seems like you would not be able to run the `appium-doctor` program at all if you didn't have Node installed in some way.